src/NJsonSchema.CodeGeneration.CSharp/CSharpGeneratorSettings.cs

```diff
@@ -121,8 +121,8 @@ public CSharpGeneratorSettings()
         /// <summary>Gets or sets a value indicating whether to use preserve references handling (All) in the JSON serializer (default: false).</summary>
         public bool HandleReferences { get; set; }
 
-        /// <summary>Gets or sets the name of a static method which is called to transform the JsonSerializerSettings used in the generated ToJson()/FromJson() methods (default: null).</summary>
-        public string JsonSerializerSettingsTransformationMethod { get; set; }
+        /// <summary>Gets or sets the name of a static method which is called to transform the JsonSerializerSettings (for Newtonsoft.Json) or the JsonSerializerOptions (for System.Text.Json) used in the generated ToJson()/FromJson() methods (default: null).</summary>
+        public string JsonSerializerSettingsOrOptionsTransformationMethod { get; set; }
```

Comment:
I think we should just use JsonSerializerSettingsTransformationMethod to avoid breaking changes and having only one config which applies for any target json library... even if the name is not 100% "clean".

Reply:
I am agree with you. Fixed)